charts/project-origin-registry/run_kind_test.sh

```diff
@@ -104,14 +107,14 @@ registries:
 EOF
 
 # install wallet
-helm install my-wallet project-origin-wallet --version 0.6.0 -f "${wallet_filename}" --repo https://project-origin.github.io/helm-registry --namespace wallet --create-namespace --wait
+helm install my-wallet project-origin-wallet --version 0.8.0 -f "${wallet_filename}" --repo https://project-origin.github.io/helm-registry --namespace wallet --create-namespace --wait
 echo "Wallet installed"
 
 # install two registries
 echo "Installing registries"
 helm install ${registry_a_name} -n ${registry_a_namespace} charts/project-origin-registry --set persistance.cloudNativePG.enabled=true,image.tag=test,service.nodePort=$registry_a_nodeport,service.type=NodePort -f "${override_values_filename}"  --create-namespace --wait
 echo "Registry A installed"
-helm install ${registry_b_name}-postfix -n ${registry_b_namespace} charts/project-origin-registry --set persistance.inMemory.enabled=true,image.tag=test,registryName=$registry_b_name,service.nodePort=$registry_b_nodeport,service.type=NodePort -f "${override_values_filename}"  --create-namespace --wait
+helm install ${registry_b_name}-postfix -n ${registry_b_namespace} charts/project-origin-registry --set transactionProcessor.replicas=1,persistance.inMemory.enabled=true,image.tag=test,registryName=$registry_b_name,service.nodePort=$registry_b_nodeport,service.type=NodePort -f "${override_values_filename}"  --create-namespace --wait
```

Comment:
I see persistance.inMemory.enabled=true. is this intentional?

Reply:
Yes, part of a full test where two registries are spun up, one inMemory, and one using postgresql to test both and how they interop. The one using inMemory, can only run a single replica for good reason, otherwise data would be inconsistent between pods.